vision/cloud-client/detect/vision_async_batch_annotate_images.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
```

Comment:
I also think we stopped adding these from comments on my last PR

Reply:
Removed